src/Type/ObjectType/CommentAuthor.php

```diff
@@ -84,7 +84,7 @@ public static function register_type() {
 							}
 
 							if ( ! empty( $args['rating'] ) ) {
-								$avatar_args['rating'] = esc_sql( $args['rating'] );
+								$avatar_args['rating'] = esc_sql( (string) $args['rating'] );
```

Comment:
This is me being lazy, since we know `$args['rating']` is a string and I didnt want to be bothered defining the entire shape of `$args`.

Reply:
👌🏻